src/middlewared/middlewared/plugins/gluster_linux/volume.py

```diff
@@ -70,6 +94,7 @@ async def common_validation(self, data, schema_name):
         Int('redundancy'),
         Bool('force'),
     ))
+    @returns(List('volumes', items=[Dict('volume', additional_attrs=True)]))
```

Comment:
Also i assume the volume is same as above - just specifying ENTRY is enough and all of these usages are covered automatically